documentation/functions/record/LOC_BUILDER_DD.md

```diff
@@ -6,13 +6,25 @@ parameters:
   - y
   - alt
   - ttl
+parameter_types:
+  label: string
+  x: number
+  y: number
+  alt: number
+  ttl: number
+---
+
+The parameter number types are as follows:
+
+```
 parameter_types:
   label: string
   x: float32
   y: float32
   alt: float32
   ttl: int
----
+```
+
 
 `LOC_BUILDER_DD({})` actually takes an object with the following properties:
```

Comment:
Can you add a `parameters_object: true` entry to any of these builders that take an object instead of a list of parameters? That way the TS types will be correct.

Reply:
How would that look?
```yaml
---
name: LOC_BUILDER_DMS_STR
parameters_object: 
  - true
parameters:
  - label
  - str
  - alt
  - ttl
parameter_types:
  label: string
  str: string
  alt: number
  ttl: number
---
```